source/code-generation/query-converter/enable-query-converter.txt

```diff
@@ -36,10 +28,25 @@ Steps
 
    .. step:: Read the user agreement
 
+   .. step:: (Optional) Review telemetry preferences
+
+      Telemetry data about the source and converted queries are 
+      temporarily stored by MongoDB to improve the query converter 
+      feature. You can disable sending telemetry data by unchecking the 
+      :guilabel:`Allow MongoDB to temporarily store source and coverted 
+      queries to improve the product feature`
+      check box.
+
+      .. note::
```

Comment:
[Question/Suggestion]

Currently having one disabling option in the note admonition makes it seem like that one has a different level of importance from the other. I really like the way you wrote this in the AI and Data Usage page with the listed bullets. Do you think we could do that here?

Reply:
Yes totally, I converted that piece to include and those bullets are now shared on each page.